CHANGELOG.md

```diff
@@ -18,6 +18,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
  
   [#478]: https://github.com/learningequality/kolibri-design-system/pull/478
 
+  - [#482]
+  - **Description:** Changed develop branch to main branch in Readme
+  - **Products impact:** Script Enhancement
```

Comment:
I'll update this to "-" before merging since this is internal to Kolibri Design System and will have not impact on products (Kolibri, Studio)